Libraries/D3D12RaytracingFallback/src/ComputeAABBs.hlsli

```diff
@@ -115,7 +115,7 @@ void main(uint3 DTid : SV_DispatchThreadID)
 
             boxData = GetBoxFromChildBoxes(leftBox, leftNodeIndex, rightBox, rightNodeIndex, outputFlag);
 
-#if COMBINE_LEAF_NODES
+#if COMBINE_LEAF_NODES_1
```

Comment:
Is "_1" intended?

Reply:
Yes, I want this to be 'commented' out. This is part of the 2nd half of the work to collapse leaf nodes, that is not done yet.